### PR TITLE
clarify how rtt & downlink are determined

### LINKED ISSUE #56: Spec should specify the layer at which rtt/downlink/effectiveType attribute is computed
NetInfo API exposes rtt/downlink/effectiveType attributes which are user agent's estimate of the network quality. The spec should specify the network layer at which these attributes should be computed.

### PR DESCRIPTION
**Pull request by igrigorik**
Closes #56, #55.

***
[Preview](https://s3.amazonaws.com/pr-preview/WICG/netinfo/clarify-rtt-downlink.html) | [Diff](https://s3.amazonaws.com/pr-preview/WICG/netinfo/01edb11...888c220.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -294,7 +294,7 @@
 
   ### <dfn>effectiveType</dfn> attribute
 
-  The `effectiveType` attribute, when getting, returns the <a>effective connection type</a> that is determined using a combination of recently observed round trip times and downlink bandwidth.
+  The `effectiveType` attribute, when getting, returns the <a>effective connection type</a> that is determined using a combination of recently observed <a href="#dom-networkinformation-rtt">rtt</a> and <a href="#dom-networkinformation-downlink">downlink</a> values.
```
Comment (igrigorik) [inline index.html]: @marcoscaceres can't get the "plain" `<a>`'s to pick up the references. Any tips for how I can avoid hard-linking this?

**Comment by igrigorik** [comment]
@tarunban ptal.

**Review by marcoscaceres — APPROVED** [review]
Fixed the xrefs for you.

**Comment by igrigorik** [comment]
@marcoscaceres thank you!

**Review by ghost — APPROVED** [review]

**Pull request merged**